synapse/tests/test_cortex.py

```diff
@@ -2383,75 +2226,6 @@ def test_cortex_splice_propdel(self):
             self.eq(len(core.eval('hehe:foo')), 0)
             self.eq(len(core.eval('hehe:baz')), 1)
 
-    def test_cortex_module_datamodel_migration_persistent(self):
```

Comment:
We should modify this test to show that we can revision dynamically loaded models rather than remove it outright